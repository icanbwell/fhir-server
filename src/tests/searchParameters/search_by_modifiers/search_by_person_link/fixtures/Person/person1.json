{
    "resourceType": "Person",
    "id": "1",
    "meta": {
        "source": "bwell",
        "security": [
            {
                "system": "https://www.icanbwell.com/access",
                "code": "bwell"
            },
            {
                "system": "https://www.icanbwell.com/owner",
                "code": "bwell"
            }
        ]
    },
    "name": [
        {
            "use": "official",
            "family": "Irving",
            "given": [
                "Lloyd"
            ]
        }
    ],
    "gender": "male",
    "birthDate": "1996-03-03",
    "link": [
        {
            "target": {
                "reference": "Patient/1",
                "type": "Patient"
            },
            "assurance": "level4"
        }
    ]
}
