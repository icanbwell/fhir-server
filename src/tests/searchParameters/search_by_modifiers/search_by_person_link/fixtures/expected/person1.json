{
    "resourceType": "Person",
    "id": "1",
    "meta": {
        "source": "bwell",
        "versionId": "1",
        "security": [
            {
                "system": "https://www.icanbwell.com/access",
                "code": "bwell"
            },
            {
                "system": "https://www.icanbwell.com/owner",
                "code": "bwell"
            },
            {
                "code": "bwell",
                "system": "https://www.icanbwell.com/sourceAssigningAuthority"
            }
        ]
    },
    "identifier": [
        {
            "id": "sourceId",
            "system": "https://www.icanbwell.com/sourceId",
            "value": "1"
        },
        {
            "id": "uuid",
            "system": "https://www.icanbwell.com/uuid",
            "value": "61abdd48-df46-5e98-ac6c-fde3cace4d07"
        }
    ],
    "name": [
        {
            "use": "official",
            "family": "Irving",
            "given": [
                "Lloyd"
            ]
        }
    ],
    "gender": "male",
    "birthDate": "1996-03-03",
    "link": [
        {
            "target": {
                "extension": [
                    {
                        "id": "sourceId",
                        "url": "https://www.icanbwell.com/sourceId",
                        "valueString": "Patient/1"
                    },
                    {
                        "id": "uuid",
                        "url": "https://www.icanbwell.com/uuid",
                        "valueString": "Patient/61abdd48-df46-5e98-ac6c-fde3cace4d07"
                    },
                    {
                        "id": "sourceAssigningAuthority",
                        "url": "https://www.icanbwell.com/sourceAssigningAuthority",
                        "valueString": "bwell"
                    }
                ],
                "reference": "Patient/1",
                "type": "Patient"
            },
            "assurance": "level4"
        }
    ]
}